### PR TITLE
website: add content-check command

### PR DESCRIPTION
**Pull request by BRKalow**
Add a `content-check` npm command and document it in the website directory's README file. This command validates the documentation content against a set of rules to ensure it will render as expected on the Developer site.

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
     "npm": ">=7.0.0"
   },
   "devDependencies": {
-    "@hashicorp/platform-cli": "^2.5.1",
-    "dart-linkcheck": "2.0.15",
```
Comment (thiskevinwang) [inline website/package.json:12]: is the removal of `dart-linkcheck` intentional?

**Review by nayres — APPROVED** [review]

**Review by thiskevinwang — COMMENTED** [review]
Left a question about removing `dart-linkcheck`.

File: website/package.json (hunk 1 above)
Comment (BRKalow) [reply website/package.json:12]: 👀 no it was not.

**Review by thiskevinwang — APPROVED** [review]
👍

**Pull request merged**